app/assets/stylesheets/orchid.scss

```diff
@@ -121,3 +121,6 @@
     content: " …";
   }
 }
+
+//uncollapse if no javascript
+.no-js .collapse { display: block; }
```

Comment:
Why was this moved to the bottom of the file? I would prefer it where it was before for CSS organization and don't see anything else in our styles that would conflict in terms of CSS precedence here. At least at a glance.

Reply:
fixed